src/modules/data/blocks/datetime/reducer.js

```diff
@@ -27,9 +27,10 @@ export const defaultEndMoment = moment().startOf( 'day' ).seconds( defaultEndTim
 
 const defaultStartDateTime = momentUtil.toDateTime( defaultStartMoment );
 const defaultEndDateTime = momentUtil.toDateTime( defaultEndMoment );
+const queryStartDate = globals.postObjects().tribe_events.tribe_start_date;
 
 export const DEFAULT_STATE = {
-	start: defaultStartDateTime,
+	start: queryStartDate ? queryStartDate : defaultStartDateTime,
```

Comment:
Doing this replaces the default start time. In your screencast, the date is correct, but it is no longer the 8:00 AM that it used to be.

Reply:
I've now included the correct start time.

Reply:
@paulmskim I've made an update here, please let me know if that's correct.